replication/README.md

```diff
@@ -366,6 +366,12 @@ message GetVolumeReplicationInfoResponse {
   // Holds the last sync time.
   // This field is REQUIRED.
   .google.protobuf.Timestamp last_sync_time = 1;
+  // Holds the last sync duration.
+  // This field is REQUIRED.
+  .google.protobuf.Timestamp last_sync_duration = 2;
+  // Holds the last sync bytes.
+  // This field is REQUIRED.
+  int64 last_sync_bytes = 3;
```

Comment:
What are "last sync bytes"? My first reading of this suggests the actual contents that were synced, but I don't think that is what you mean. Is it the number of bytes transferred with the last synchronization, maybe?

Reply:
yes it is that.

Reply:
can you also add a comment like below
`// The value of this field MUST NOT be negative.`